src/test/java/org/opensrp/web/rest/SearchResourceTest.java

```diff
@@ -98,6 +94,7 @@ public void shouldSearchClient() throws ParseException {
 		mockHttpServletRequest.addParameter("alt_name", firstName);
 		mockHttpServletRequest.addParameter("attribute", "next_contact_date:2022-06-15");
 		mockHttpServletRequest.addParameter("dob", String.valueOf(birthDate));
+		mockHttpServletRequest.addParameter("identifier", "fsdf"+":"+ "sfdf");
```

Comment:
Please update the test to check for the alt phonenumber too

Reply:
Done

Reply:
Update the tests to assert that at least one of the clients returned has the alt phone number in the search parameters.